cmd/add_subnet_validator.go

```diff
@@ -124,6 +110,9 @@ func createSubnetValidatorFunc(cmd *cobra.Command, args []string) error {
 	println()
 	for _, nodeID := range info.nodeIDs {
 		ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
+		valInfo := info.valInfos[nodeID]
```

Comment:
Should we add 

https://github.com/ava-labs/subnet-cli/blob/26d27758816ae9253e7260701cc125ccc72d2dd8/cmd/wizard.go#L142-L156

as in `wizard` command?

It's only populated there, but not in `subnet-validator` sub-command

Reply:
Yeah, I will. I was on the fence about it but it makes sense.